### PR TITLE
Add room request fake data

### PR DESCRIPTION
**Pull request by jam-wa**
thanks to nina 😻

### CODE REVIEW

File: app/models/room_request.py [hunk 1]
```diff
@@ -92,10 +101,24 @@ def __repr__(self):
...
-#    pass
\ No newline at end of file
+            seed()
+            for i in range(count):
+                rr = RoomRequest(
+                        first_name = fake.first_name(),
+                        last_name = fake.last_name(),
+                        email = fake.email(),
+                        patient_full_name = fake.full_name(),
+                        **kwargs)
```
Comment (katiejiang) [inline app/models/room_request.py]: I think we should add fake data for all the columns! That way we can more easily test front-end and we don't have to manually enter this data everytime through `**kwargs`.

**Review by katiejiang — CHANGES_REQUESTED** [review]

File: app/models/room_request.py [hunk 2]
```diff
@@ -92,10 +101,56 @@ def __repr__(self):
...
+                        address_line_one = fake.street_address(),
+                        address_line_two = fake.secondary_address(),
+                        city = fake.city(),
+                        state = fake.state(),
+                        zip_code = fake.zipcode(),
+                        country = fake.country(),
+                        primary_phone = fake.phone_number(),
+                        secondary_phone = fake.phone_number(),
+                        email = fake.email(),
+                        primary_language = random.choice(["English", "Spanish""]),
```
Comment (katiejiang) [inline app/models/room_request.py]: You have an extra quotation mark here!

File: app/__init__.py [hunk 3]
```diff
@@ -37,7 +37,7 @@ def create_app(config):
     app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
     #Connected to Azure database
     param_string = "DRIVER={};SERVER={};DATABASE={};UID={};PWD={}".format(
-    "{SQL Server}",
+    "{ODBC Driver 17 for SQL Server}",
```
Comment (katiejiang) [inline app/__init__.py]: Why is this getting changed?

Comment (jam-wa) [reply app/__init__.py]: azure...??

**Pull request merged**